domain/src/main/java/org/apache/rocketmq/eventbridge/domain/cache/EventSourceCacheService.java

```diff
@@ -0,0 +1,96 @@
+ /*
+  * Licensed to the Apache Software Foundation (ASF) under one or more
+  * contributor license agreements.  See the NOTICE file distributed with
+  * this work for additional information regarding copyright ownership.
+  * The ASF licenses this file to You under the Apache License, Version 2.0
+  * (the "License"); you may not use this file except in compliance with
+  * the License.  You may obtain a copy of the License at
+  *
+  *     http://www.apache.org/licenses/LICENSE-2.0
+  *
+  * Unless required by applicable law or agreed to in writing, software
+  * distributed under the License is distributed on an "AS IS" BASIS,
+  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  * See the License for the specific language governing permissions and
+  * limitations under the License.
+  */
+
+package org.apache.rocketmq.eventbridge.domain.cache;
+
+import org.apache.commons.lang3.StringUtils;
+import org.apache.rocketmq.eventbridge.domain.model.bus.EventBus;
+import org.apache.rocketmq.eventbridge.domain.model.source.EventSource;
+import org.apache.rocketmq.eventbridge.domain.repository.EventBusRepository;
+import org.apache.rocketmq.eventbridge.domain.repository.EventSourceRepository;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.cache.Cache;
+import org.springframework.cache.CacheManager;
+import org.springframework.cache.annotation.CacheConfig;
+import org.springframework.cache.annotation.Cacheable;
+import org.springframework.stereotype.Component;
+
+import javax.annotation.PostConstruct;
+import java.util.List;
+
+import static org.apache.rocketmq.eventbridge.domain.cache.CacheName.EVENT_SOURCE;
+
+/**
+ * @Author changfeng
+ * @Date 2022/4/25 11:15 上午
+ */
+@Component
+@CacheConfig(cacheNames = EVENT_SOURCE, cacheManager = "cacheManager")
+public class EventSourceCacheService {
```

Comment:
EventSourceCacheService and EventSourceService should not be separated.

Reply:
EventSourceCacheService has been deleted, and the related method is placed in HTTPEventSourceService.